gson/src/main/java/com/google/gson/FieldNamingStrategy.java

```diff
@@ -37,4 +40,16 @@ public interface FieldNamingStrategy {
    * @since 1.3
    */
   public String translateName(Field f);
+
+  /**
+   * Translates the field name into its JSON field alternative names representation. Used for
```

Comment:
Perhaps write something like this:

> Returns alternative names for this field when it is being deserialized. This is similar...

Reply:
Javadoc has been updated as per the suggestion.